src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -1378,9 +1378,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return _lazyLexicalSortKey
         End Function
 
+        Public Function LocationKey(ByVal l As Location) As Long
```

Comment:
We already have LexicalSortKey class and Compilation.CompareSourceLocations method that utilizes it. Consider taking advantage of them so that we get consistent sorting throughout the compiler. For example, GetBestName method in this class uses LexicalSortKey.